examples/c/components/http-hello-world/wasmcloud.toml

```
@@ -0,0 +1,8 @@
+name = "http-hello-world"
+version = "0.1.0"
+language = "c"
+type = "component"
+
+[component]
+wit_world = "hello"
```

```suggestion
[component]
build_command = "make http_hello_world.wasm"
build_artifact = "http_hello_world.wasm"
wit_world = "hello"
```

This small change should let folks run `wash build` to build and sign it!

---

Awesome!